xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -251,7 +251,7 @@ bool CAndroidUtils::SetNativeResolution(const RESOLUTION_INFO &res)
     CXBMCApp::SetDisplayMode(atoi(res.strId.c_str()));
     s_res_cur_displayMode = res;
   }
-  else if (abs(currentRefreshRate() - res.fRefreshRate) > 0.0001)
+  else if (std::abs(currentRefreshRate() - res.fRefreshRate) > 0.0001)
```

Comment:
Doesn't `std::abs` have a float version (http://en.cppreference.com/w/cpp/numeric/math/fabs)?
Should I include `cmath` in this file also it currently compiles?